storage.go

```diff
@@ -564,7 +625,7 @@ func (f *file) Download(filePath string) ([]byte, error) {
 			panic(err)
 		}
 
-		if resErr.Status == "404" {
+		if resErr.StatusCode == "404" {
```

Comment:
```suggestion
		if resErr.StatusCode == http.StatusNotFound {
```

Reply:
The API returns statusCode in the response body as a string, which is why it is unmarshalled into a string field. I will write a custom unmarshaler in a separate PR.